src/syft/core/node/domain/service/request_handler_service.py

```diff
@@ -213,13 +213,18 @@ def _object2proto(self) -> GetAllRequestHandlersResponseMessage_PB:
             the other public serialization methods if you wish to serialize an
             object.
         """
+        # For handler["created_time"], it's a large number. In order to keep it's precision
```

Comment:
I don't think we need such precision + 64 bits might be enough? Can't we make this time relative to the process start or the current time (only seconds precision should be enough). If not, encode it as a string directly?

Reply:
`Float` uses a 32-bit float type in proto message. And only 24-bit are used for precision. So a value of `time.time()` will loss some precision. Please try the following and you will see it:
```python
t = time.time()
ptr = sy.lib.python.Float(t).send(duet)
print(t)
print(ptr.get())
```
But add and sub something like 1.6e9 truly looks ugly. Yes, encode it as str is a good idea, I will do that.